src/rdkafka_cgrp.c

```diff
@@ -1064,8 +1067,17 @@ static void rd_kafka_cgrp_handle_JoinGroup (rd_kafka_t *rk,
                         rd_kafka_cgrp_assignor_handle_Metadata_op);
                 rd_kafka_op_set_replyq(rko, rkcg->rkcg_ops, NULL);
 
-                rd_kafka_MetadataRequest(rkb, &topics,
-                                         "partition assignor", rko);
+                rd_kafka_MetadataRequest(
+                        rkb, &topics,
+                        "partition assignor",
+                        /* cgrp_update=false:
+                         * Since this Metadata request may not be identical
```

Comment:
I was confused around here for a while. This comment is fine... but if it had read "Since the subscription list across all members may not be identical to this consumer's ...", i probably would have spent less time confused.